src/reference/asciidoc/whats-new.adoc

```diff
@@ -35,6 +35,17 @@ For more information, see <<security>>.
 The `FileSplitter`, which splits text files into lines, was added in 4.1.2.
 It now has full support in the `int-file:` namespace; see <<file-splitter>> for more information.
 
+[[x4.2-zk]]
+==== Zookeeper Support
+
+Zookeeper support has been added to the framework to assist when running on a clustered/multi-host environment.
+
+* ZookeeperMetadataStore
+* ZookeeperLockRegistry
+* Zookeeper Leadership (start/stop adapters based on leadership).
```

Comment:
??? Are you going to add more info here to each component?
And there is need to say something like:

> The following components are presented in the Spring Integration Zookeeper module now: